### DISCUSSION TITLE
很有参考价值的局面测试一例，看看引擎用多久给出正解？

### DISCUSSION THREAD

**Discussion by ATH139** [official-pikafish/Pikafish#56] [General]
4kab2/4a4/b2r5/2RN4p/2P6/3C2p2/P3P1c1P/1r2B1n2/1c7/1RBAKA3 b
皮卡鱼引擎对于此局面是可以解出正招：即炮2平9！（关闭云库测试）
但引擎版本更新不同，计算时间也不同，有快有慢甚至个别版本解不出正招！欢迎对比测试~

**Comment by otischan-dev** [comment]
0.4秒已經算出炮2平9 雖然沒到9999/29999分
<img width="1267" height="590" alt="image" src="https://github.com/user-attachments/assets/395bc141-d145-4e7b-a19d-690aaacf1415" />
